### DISCUSSION TITLE
Getting quotes and lsratio from different exchanges

### DISCUSSION THREAD

**Discussion by andreltr** [serkor1/cryptoQuotes#21] [Q&A] [answered]
Hello,

This question may be out of the scope of this group but I take the freedom in making it.
Suppose I want to get the largest number of "complete records" possible.  According to the wiki this combination would be to get spot rows from kucoin and futures from kraken, respectively.  

Each exchange provides 5000 records for spot from  kucoin and 1500 records for futures from kraken.

My question is:  index is the same for, say, the last 1500 records of both quotes and lsratio data obtained?
TIA.

Best.

**Comment by serkor1** [comment, marked as answer]
Hi @andreltr, 

Feel free to ask away - it also helps me get an idea of how people use {cryptoQuotes}.

All indexes are the samme across exchanges assuming that you you specify the same _interval_ in each function.

Please be aware that you can extract records beyond the limits listed in the wiki. The limits are for single calls only, you can for most of the _intervals_ extract OHLC-V data way back to the listing date.

Hope this helps,

Serkan
